cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -350,4 +358,6 @@ func init() {
 	depositNotaryCmd.Flags().String(walletAccountFlag, "", "Wallet account address")
 	depositNotaryCmd.Flags().String(refillGasAmountFlag, "", "Amount of GAS to deposit")
 	depositNotaryCmd.Flags().String(notaryDepositTillFlag, "", "Notary deposit duration in blocks")
+	RootCmd.AddCommand(netmapCandidatesCmd)
```

Comment:
```suggestion
	
	RootCmd.AddCommand(netmapCandidatesCmd)
```
? all they are separated by an empty line